tests/server_selection/framework.rs

```diff
@@ -1,7 +1,6 @@
-use mongodb::{Client, ThreadedClient};
+use mongodb::Connector;
```

Comment:
Changes to this file are a mix of using the updated API, the renaming of `StreamConnector`, and from rustfmt.